tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+        "outDir": "./dist",
+        "pretty": true,
+        "target": "es5",
+        "lib": [
+          "es2017"
+        ],
+        "module": "commonjs",
+        "allowJs": true,
+        "sourceMap": true,
+        "strict": true,
+        "noImplicitAny": false // Needed to compile tap and ansi-escapes
+    },
+    "include": [
+        "./src/**/*",
+    ]
+}
```

Comment:
tiny thing, but you should turn on `"files.insertFinalNewline": true,` in vscode to prevent this

Reply:
Okay, you'll have to show me where to do this though :)